portugues/ladrao/res/bruce.md

```diff
@@ -0,0 +1,34 @@
+                            __--~~--_     _--~~--__
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.